piqueserver/player.py

```diff
@@ -374,3 +410,14 @@ def timed_out(self):
 def encode_lines(value):
     if value is not None:
         return [encode(line) for line in value]
+
+# calculate the distance between an object with x, y, and z members and a
+# tuple containing (x, y, z) coords.
+def distance(a, b):
```

Comment:
Very sure the Vertex3 object has this method already...

Reply:
Yes, it does... but not directly. It implements subtraction and length. But that ammounts to the same thing.